tests/integration-tests/tests/test_efa/test_efa/test_efa/sge_osu_submit.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#$ -pe mpi 144
```

Comment:
Having the number of slots statically set to 144 can be a problem with some instance types, especially if they have more than 144 vcpus. Same in the other script.